### DISCUSSION TITLE
抱歉大佬，我想请求删除最近4条discussion

### DISCUSSION THREAD

**Discussion by RookieOHY** [pinky-pig/arvin#13] [General]
大佬 我fork了你博客仓库想学习下 由于我对giscus不熟悉，忘记修改配置文件，导致我的**测试评论转发到您这里**了，怕污染您的博客评论区，故申请删除最近4条discussion。打扰到您了，实在抱歉哈！ @pinky-pig

**Comment by 6owen** [comment]
哈哈，没事，代码可以免费随便用，别忘了 star 🌟。
不过网站要是添加上一行小字 `设计来自于 [Arvin](https://github.com/pinky-pig/arvin/)`就更加感谢了🥹

**Reply by RookieOHY** [reply]
会的会的 fork的readme 和 站点都会加上作者来源！！
